charts/osdfir-infrastructure/charts/openrelik/templates/_initContainer.tpl

```diff
@@ -0,0 +1,35 @@
+{{/*
+Init Container for when a OpenRelik server starts. Required for setting up
+OIDC when enabled due to the changing needing to be in settings.toml
+*/}}
+{{- define "openrelik.initContainer" -}}
+- name: init-openrelik
```

Comment:
Nice resolution using and init container for now, let's hope this can be refactored out in the future.